### DISCUSSION TITLE
Frames, covers, and machines.

### DISCUSSION THREAD

**Discussion by Chinthor** [MrTJP/ProjectRed#1689] [Q&A]
Just trying to clear up some confusion on my part. Have seen ProjectRed included or available to add in many modpacks, and see the latest version is for 1.15+ and 1.16+. Great. But does a lot of it work? It looks like frames and frame motors have not been implemented, and in every install I've tried so far I cannot craft microparts like covers or panels.

So is there a 1.16.5 compatible configuration that has some of these features? Or is it well known that these don't work yet and I'm just not finding the documentation?

Thanks

PS: If not, what is the latest version combination that would allow for moving structures and flying mining machines?

**Comment by pat2rome** [comment]
Not sure about if frames are implemented or not, since I haven't played 1.16 yet, but can you craft microblocks from vanilla blocks? If you can, the reason some blocks don't work is they have to be manually added to the microblocks.cfg file. If you can't craft microblocks from something like stone or oak wood planks, then it's likely that isn't fully implemented yet either.

**Comment by MrTJP** [comment]
Frames and motors are now part of Project Red since 1.18 release (https://github.com/MrTJP/ProjectRed/discussions/1775)
